tests/Routing/UrlGeneratorTest.php

```diff
@@ -364,4 +356,32 @@ private function getGenerator($prependLocale = false, $returnArgument = 1, $useA
             $prependLocale
         );
     }
+
+    /**
+     * @param array $expectedParameters
+     * @param       $expectedRoute
+     * @param int   $referenceType
+     *
+     * @return \PHPUnit_Framework_MockObject_MockObject|UrlGeneratorInterface
+     */
+    private function expectRouter(
```

Comment:
CS: no line breaks in method definitions. 😄

Reply:
Fixed in 88d9d28703ef82958c464bb4909698b0eb206ced